fsociety/web_apps/cli.py

```diff
@@ -9,3 +9,6 @@
 
 def cli():
     tools_cli(__name__, __tools__)
+
+
+__name__ = "web_apps"
```

Comment:
I am not sure we should override the `__name__` built-in like this. We can easily parse the `fsociety.` out.

Reply:
ah, i guess i wasn't aware of that working that way. i'll refactor this back out